src/plugins/mathcheck/README.md

```diff
@@ -26,31 +26,56 @@ Keynames are all either relative to to-be-tested key (starting with `./` or `../
 `check/math = "<= - @/testval1 * @/testval2 @/testval3"` tests if the keyvalue is less than or equal to testval1 - (testval2 * testval3) and yields an error if not.
 
 Full example:
-
-    kdb mount mathcheck.dump /example/mathcheck dump mathcheck
-    kdb set user/example/mathcheck/a 3.1
-    kdb set user/example/mathcheck/b 4.5
-    kdb set user/example/mathcheck/k 7.6
-    kdb setmeta user/example/mathcheck/k check/math "== + ../a ../b"
-    kdb set user/example/mathcheck/k 7.7   # fails
-
+```sh
+# Backup-and-Restore:/examples/mathcheck
+kdb mount mathcheck.dump /examples/mathcheck dump mathcheck
+kdb set /examples/mathcheck/a 3.1
+kdb set /examples/mathcheck/b 4.5
+kdb set /examples/mathcheck/k 7.6
+kdb setmeta user/examples/mathcheck/k check/math "== + ../a ../b"
+# should fail
```

Comment:
Yes, we actually need `#` for key names, so you need to pass it to the shell anyway.

My question was more about if you pass it to the shell correctly or if such a `#` somehow messes up the tool.